js/index.js

```diff
@@ -1,7 +1,124 @@
+
+Integer finibus enim non quam rutrum dignissim. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vivamus consectetur vulputate felis et blandit. Vestibulum vestibulum metus id leo lacinia pharetra. Aenean eu elit blandit, dignissim ligula consequat, pulvinar nibh. Praesent eget ipsum risus. Ut nec pretium tellus. Maecenas porttitor posuere ultricies. Etiam vel porta nibh. Sed pulvinar leo cursus commodo rutrum. Morbi a vestibulum est, in ullamcorper odio. Ut porta volutpat est, non pellentesque lectus venenatis vel. Etiam hendrerit sapien ac pulvinar viverra.
+
+Ut ac ornare urna. Aliquam non suscipit odio. Nullam congue neque in sem blandit, imperdiet finibus sapien consequat. Nullam tristique pharetra justo a egestas. Nunc rutrum interdum urna, ut ullamcorper lectus blandit non. Pellentesque bibendum dolor sem, pharetra convallis mi consequat at. Aliquam id nulla eu nulla tempus eleifend. Aliquam posuere non dolor sed fermentum. Sed sed efficitur neque, eget mattis nunc. Nulla condimentum quam eget eleifend hendrerit. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Ut porttitor tempus mi, et luctus neque congue non. Nam ut egestas augue, at sagittis lacus. Nulla consequat metus quis scelerisque sodales. `;
+
+let numWords = 0;
+
+if (lorem[0]) numWords = 1;
+for (let i = 0; i < lorem.length; i++) {
+  if (lorem[i] === ` `) {
+    while (lorem[i] == ` `) {
+      i++;
+    }
+    if (i < lorem.length) numWords++;
+  }
+}
+console.log(`This text contains ${numWords} words`);
+
+let numEt = 0;
+
+for (let i = 0; i < lorem.length - 1; i++) {
+  if (lorem[i] === `e` && lorem[i + 1] === `t`) numEt++;
+}
+console.log(`This text contains the latin word "et" ${numEt} times`);
+
+// Bonus 2:
+
+let phraseToCheck = "Was it a car -- or a cat I saw?";
 
-// Iteration 3: Loops
\ No newline at end of file
+j = phraseToCheck.length - 1;
+for (i = 0; i <= j; i++) {
+  if (j === i) console.log(`Your phrase "${phraseToCheck}" is a palindrome`);
+  while (
+    (phraseToCheck[i].toUpperCase() > "Z" ||
+      phraseToCheck[i].toUpperCase() < "A") &&
+    i < phraseToCheck.length - 1
+  ) {
+    i++;
+  }
+  while (
+    (phraseToCheck[j].toUpperCase() > "Z" ||
+      phraseToCheck[j].toUpperCase() < "A") &&
+    j > 0
+  ) {
+    j--;
+  }
+  if (
+    i === j ||
+    (i === j - 1 &&
+      phraseToCheck[i].toUpperCase() === phraseToCheck[j].toUpperCase())
+  )
+    console.log(`Your phrase "${phraseToCheck}" is a palindrome!`);
+  else if (phraseToCheck[i].toUpperCase() != phraseToCheck[j].toUpperCase()) {
+    console.log(`Your phrase "${phraseToCheck}" is not a palindrom`);
+    i = j + 1;
+  }
+  j--;
+}
```

Comment:
Could be simplified, but cleaning first the 
Doesn't work with "Amor, Roma" because of the comma. You could clean the phraseToCheck beforehand from any special characters, do the same with the reversed phraseToCheck, and then compare the 2.